testpilot/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -91,6 +91,13 @@ export default `
                 </section>
               </div>
               <div class="details-list"></div>
+              <div data-hook="active-user">
```

Comment:
I realize duping this hook is a hack, but it will appease legal and handle the edge case